api/v1alpha1/clusterextension_types.go

```diff
@@ -140,6 +144,7 @@ type ClusterExtensionInstallConfig struct {
 	// the ServiceAccount provided via this field should be configured with the
 	// appropriate permissions to perform the necessary operations on all the
 	// resources that are included in the bundle of content being applied.
+	//+kubebuilder:validation:Required
 	ServiceAccount ServiceAccountReference `json:"serviceAccount"`
 
 	// preflight is an optional field that can be used to configure the preflight checks run before installation or upgrade of the content for the package specified in the packageName field.
```

Comment:
Do we need to be more descriptive about what the default preflight checks are?

Context: https://github.com/openshift/api/pull/2067/files#r1812435988